embedding/cpp/elm_flutter_view.cc

```diff
@@ -45,11 +45,23 @@ bool ElmFlutterView::RunEngine() {
   }
 
   evas_object_ =
-      static_cast<Evas_Object *>(FlutterDesktopViewGetEvasObject(view_));
+      static_cast<Evas_Object*>(FlutterDesktopViewGetEvasObject(view_));
   if (!evas_object_) {
     TizenLog::Error("Could not get an Evas object");
     return false;
   }
+
+  evas_object_event_callback_add(
+      evas_object_, EVAS_CALLBACK_DEL,
```

Comment:
When does the object get deleted? By who?

Reply:
evas_object_ is deleted when parent_ is deleted.
This deletion is process by Eo of EFL. (ElmFlutterView always needs a valid evas object (parent).)
Of course, if the user deletes FlutterView, I think that another engine problem(crash?) may occur.
I am considering several cases.